sysutils/borgbackup/Portfile

```diff
@@ -70,4 +71,8 @@ post-destroot {
         ${destroot}${zsh_compl_path}
 }
 
+variant fuse description {FUSE support} {
+    depends_lib-append      port:py${python.version}-llfuse
+}
+
```

Comment:
Is this actually sufficient to enable FUSE support in the port without any build changes? Also, don't you need to depend on the OS X fuse implementation (osxfuse I think?)

Reply:
> Is this actually sufficient to enable FUSE support in the port without any build changes? 

I believe so. I tested the functionality that requires FUSE (`borg mount`) and it worked.

> Also, don't you need to depend on the OS X fuse implementation (osxfuse I think?)

`osxfuse` is a dependency of `py-llfuse`.

Reply:
The problem is what if the user already has pyXY-llfuse installed and does not select this variant? It looks like support for FUSE will get built anyway even if not requested. Can you figure out how to disable the support unless it is requested by this variant?

Reply:
[py-llfuse is a required dependency of borgbackup](https://github.com/borgbackup/borg/blob/master/setup.py#L56), and without it the `borg mount` and `borg unmount` commands will not work. I don’t think we want to forcibly break those commands for users that already have `pyXY-llfuse`. If those users don’t have any use for `borg (un)mount` they can just not use those commands. 

The outstanding question is why this dependency was removed in fa21d92, which hopefully @lpefferkorn can chime in on. The only reason I can think of is that osxfuse is fairly invasive, and apart from the 2 aforementioned commands, borg functions perfectly well without it (and the `pyXY-llfuse` library). Perhaps we could make `+fuse` the default variant, and then users that don’t want to deal with osxfuse can install with `-fuse`.

Reply:
Unless fuse support can be disabled, it needs to be added back as a dependency then.